button.js

```diff
@@ -10,27 +10,48 @@ function slugify(text) {
     .replace(/_/g, '-');
 }
 
-const repos = ['hip-new'];
+const repos = [
+  { name: 'hip-new', domain: 'www-hip-new' },
+  { name: 'hip-microapp-directory', domain: 'hip-microapp-directory' },
+  { name: 'hip-message-service', domain: 'hip-message-service' },
+  { name: 'platform-search-service', domain: 'platform-search-service' },
+  { name: 'lectio', domain: 'lectio' },
+  { name: 'getquote.com.au', domain: 'getquote.com.au' },
+  { name: 'socium', domain: 'socium' },
+  { name: 'opus', domain: 'opus' },
+  { name: 'conmendator', domain: 'conmendator' }
```

Comment:
I don't like the idea of this..... a static list that needs to be maintained.... can not make it dynamic somehow?... and just handle the specific edge of hip-new.....